src/Chapter1.hs

```diff
@@ -631,8 +640,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+  let absN = abs n
+      last1 = absN `mod` 10
+      last2 = (absN `mod` 100) `div` 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both `mod` and `div` of ten. 

The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(x, y) = divMod absN 10
```

You can see how we could pattern match on the pair 🙂

Reply:
I haven't found that kind of function in the other language stdlib. Somehow reading the Prelude documentation is pretty intimidating for me, need to practice more.

Reply:
Don't worry about that! The Prelude docs are not in the best shape at the moment and currently, the work is going on in that direction, so I hope it would be much more approachable in the future. But for now, don't hesitate to ask about anything here 🙂